frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -161,9 +166,15 @@ def _create_page(self, obj):
         db_page.fingerprint = obj.meta['fingerprint']
         db_page.state = Page.State.NOT_CRAWLED
         db_page.url = obj.url
-        db_page.depth = 0
         db_page.created_at = datetime.datetime.utcnow()
         db_page.meta = obj.meta
+
+        if not isinstance(obj, frontera_response):
```

Comment:
I'm not sure if we need to store *response* headers and cookies. I think we should just expect that ``obj`` could be a response, and we should set headers probably from ``obj.request.headers`` in this case.